### PR TITLE
Replaced ``` code blocks with Code Toggler

### PR DESCRIPTION
**Pull request by tilsammans**
Hope this is appreciated. I can help improve the documentation in lots of other areas, too.

### CODE REVIEW

File: content/en/hosting-and-deployment/hosting-on-netlify.md [hunk 1]
```diff
@@ -65,10 +65,10 @@ You can [set Hugo version](https://www.netlify.com/blog/2017/04/11/netlify-plus-
```
Comment (jmooring) [inline content/en/hosting-and-deployment/hosting-on-netlify.md:65]: To the best of my knowledge, Netlify abandoned their plans to support anything other than TOML.

File: content/en/news/http2-server-push-in-hugo.md [hunk 2]
```diff
@@ -23,7 +23,8 @@ If you navigate to https://gohugo.io and look in the Chrome developer network co
 ## 1. Configure Netlify Output Formats
 
 Add a new custom media type and two new output formats to `config.toml`. For more on output formats in Hugo, see [Custom Output Formats](/templates/output-formats/).
```
Comment (jmooring) [inline content/en/news/http2-server-push-in-hugo.md:25]: Why not use a code toggle here as well?

File: content/en/hosting-and-deployment/hosting-on-netlify.md (hunk 1 above)
Comment (tilsammans) [reply content/en/hosting-and-deployment/hosting-on-netlify.md:65]: Wasn't aware. I will revert back to only TOML for Netlify-specific config.

File: content/en/news/http2-server-push-in-hugo.md (hunk 2 above)
Comment (tilsammans) [reply content/en/news/http2-server-push-in-hugo.md:25]: Will add.

**Pull request merged**